portfolio/src/main/webapp/script.js

```diff
@@ -11,11 +11,15 @@ function getRandomFact() {
   factContainer.innerText = fact;
 }
 
-function getGreeting() {
+function getComment() {
   fetch('/data')
-  .then(response => response.text())
-  .then((greeting) => {
-    const greetingContainer = document.getElementById('greeting-container');
-    greetingContainer.innerHTML = greeting;
-  })
+  .then(response => response.json())
+  .then((comments) => {
+    const commentContainer = document.getElementById('comment-container');
+    let styledComments = "";
+    for (let i = 0; i < comments.length; i++) {
```

Comment:
consider using [a for-of loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) here, if possible:
https://google.github.io/styleguide/jsguide.html#features-for-loops